ark-activemq/values.yaml

```diff
@@ -0,0 +1,99 @@
+# Default values for ark-activemq.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Please leave the `replicaCount` to 1 for the time being. Clustering is not
+# supported yet.
+replicaCount: 1
+
+image:
+  repository: 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_activemq
+  pullPolicy: IfNotPresent
+  tag: latest
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Console username and password: Please set a secure password
+adminUsername: admin
+adminPassword: admin
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+# Uncomment ports that you want ActiveMQ to open. You can also pass the ports
+# you want to open as arguments to the `helm` program. Please note you need to
+# open at least one of the following ports: amqp, mqtt, openwire, websocket or
+# stomp.
+ports:
+  metrics: 5556
+#  console: 8161
+#  amqp: 5672
+#  mqtt: 1883
+#  openwire: 61616
+#  websocket: 61614
+#  stomp: 61613
+serviceType: ClusterIP
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+readinessProbe:
+  initialDelaySeconds: 10
+  periodSeconds: 5
+  timeoutSeconds: 2
+  failureThreshold: 2
+
+livenessProbe:
+  initialDelaySeconds: 10
+  periodSeconds: 5
+  timeoutSeconds: 2
+  failureThreshold: 2
+
+# The persistent volume claim is managed outside this Helm chart. Please set
+# `existingClaim` to the correct value for your deployment.
+persistence:
+  existingClaim: activemq-pvc
+  mountPath: /app/data
+  subPath:
+
```

Comment:
I don't understand why we should define PVC outside of helm . As both On-Prem cluster and cloud, we need to define the PVC object.  What advantage we can have if we do though deploy script I assume with Helm or though the script we  create PVC object only. Please correct if i am wrong.

Reply:
@naveen-armedia We already discussed this. That's because if I define the PVC inside the Helm charts as, say, backed by EBS, then the Helm chart can only be deployed on AWS and not on premises (because EBS is not available in a non-AWS platform). 
I just added a comment in [this Confluence page](https://arkcase.atlassian.net/wiki/spaces/AANTA/pages/1431666701/Helm+chart+standards+and+best+practices) to that effect. Please comment there if you want to discuss it.

Reply:
Sorry @fabricetriboix I could not able to understand rationality of keeping PVC object definition outside helm.  May be after our meeting on deploy script would clear my thoughts. After our meeting I will keep my questions in the confluence page.Thanks.

Reply:
@naveen-armedia OK, cool!